### DISCUSSION TITLE
1.5.5

### DISCUSSION THREAD

**Discussion by stnguyen90** [appwrite/appwrite#8027] [Show and tell]
### Notable changes

* Change SMS verification message to only have the code by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7912
* Add new country `Taiwan` with its translations by @xuelink in https://github.com/appwrite/appwrite/pull/7873
* Add Hong Kong (HK) to countries list by @deanurag in https://github.com/appwrite/appwrite/pull/7962
* Add French Polynesia flag to flags.php by @PineappleIOnic in https://github.com/appwrite/appwrite/pull/8007
* Enable auto upgrade for mariadb container by @stnguyen90 in https://github.com/appwrite/appwrite/pull/8020

## Fixes

* Use team internal ID for checks and queries for membership by @lohanidamodar in https://github.com/appwrite/appwrite/pull/7836
* Use internal IDs for queries and checks by @lohanidamodar in https://github.com/appwrite/appwrite/pull/7839
* Remove redundant commas by @jgentes in https://github.com/appwrite/appwrite/pull/7764
* Remove a redundant call to fetch the topic document again by @ItzNotABug in https://github.com/appwrite/appwrite/pull/7894
* Fix wrong refresh var for Autodesk by @eldadfux in https://github.com/appwrite/appwrite/pull/7897
* Fix email attachment example by @gewenyu99 in https://github.com/appwrite/appwrite/pull/7681
* Add missing chunkId param to create file abuse key by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7913
* Fix delete message event not firing by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7906
* Fix worker crash when using custom SMTP provider by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7915
* Update email attachments param by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7885
* Fix MFA protected group by @Meldiron in https://github.com/appwrite/appwrite/pull/7947
* Fix recovery code removal by @Meldiron in https://github.com/appwrite/appwrite/pull/7950
* Add recovery code to List factors by @Meldiron in https://github.com/appwrite/appwrite/pull/7949
* Fix challenge type check by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7981
* Fix MFA links in specs by @gewenyu99 in https://github.com/appwrite/appwrite/pull/7966
* Add missing 'apis' attribute to projects collection by @stnguyen90 in https://github.com/appwrite/appwrite/pull/7997
* Update user create error message for console to be console specific by @gewenyu99 in https://github.com/appwrite/appwrite/pull/7996
* Add DB environment variables to appwrite-worker-mails by @mbackonja in https://github.com/appwrite/appwrite/pull/8002
* Delete related attributes on delete collection  by @fogelito in https://github.com/appwrite/appwrite/pull/7985
* Fix server errors from invalid or outdated cookies by @PineappleIOnic in https://github.com/appwrite/appwrite/pull/8008
* Fix delete MFA authenticator response model by @stnguyen90 in https://github.com/appwrite/appwrite/pull/8005
* Fix MFA with admin mode by @TorstenDittmann in https://github.com/appwrite/appwrite/pull/7984

## Miscellaneous

* Update getEnv to use system lib by @eldadfux in https://github.com/appwrite/appwrite/pull/7895
* Update SDK and docs links in readme by @gewenyu99 in https://github.com/appwrite/appwrite/pull/7978
* Update README.md by @LauraDuRy in https://github.com/appwrite/appwrite/pull/6358
* Bump console to version 4.0.6 by @stnguyen90 in https://github.com/appwrite/appwrite/pull/8017

> Note: please run the [migrate command](https://appwrite.io/docs/upgrade#migration) even if upgrading from 1.5.x.

## New Contributors
* @jgentes made their first contribution in https://github.com/appwrite/appwrite/pull/7764
* @ItzNotABug made their first contribution in https://github.com/appwrite/appwrite/pull/7894
* @xuelink made their first contribution in https://github.com/appwrite/appwrite/pull/7873
* @LauraDuRy made their first contribution in https://github.com/appwrite/appwrite/pull/6358
* @mbackonja made their first contribution in https://github.com/appwrite/appwrite/pull/8002
* @deanurag made their first contribution in https://github.com/appwrite/appwrite/pull/7962

**Full Changelog**: https://github.com/appwrite/appwrite/compare/1.5.4...1.5.5

<hr /><em>This discussion was created from the release <a href='https://github.com/appwrite/appwrite/releases/tag/1.5.5'>1.5.5</a>.</em>

**Comment by aabmets** [comment]
> Add new country Taiwan with its translations 

Is Appwrite now cancelled in China?
